addon/components/table-fluid/component.js

```diff
@@ -42,6 +46,22 @@ export default EmberCollection.extend(SlotsMixin, EKMixin, {
     }
   }),
 
+  _2: observer('errors', function () {
+    if ((this.errors || []).length > 0) {
+      if (this.errors[0].message === 'limit_exceeded') {
+        this.toast.info(
+          `${this.intl.t('errors.402.limit_exceeded.congratulation')} ${this.intl.t(
+            'errors.402.limit_exceeded.description',
```

Comment:
should we have a dedicated message ? The `congratulation` is a bit funky IMO

Reply:
I agree with Thomas.
Also, wasn't this suppose to be an extra "row" element at the bottom of the table ? Did we settle for the toast instead ?

Reply:
I see with Simon for the message and the use of the toast. Do you want change that @SimonBoxus or it's ok for you?

Reply:
@tgallice We have re-used the same message displayed when users reach their search limit (modal)
@Miexil I was not informed about the extra "row", after discussing with @JulienVannier66, I told him to add an alert with the error message. Actually, the alert is displayed when user reach his limit, it fix the isssue and I don't want to to change everythings :) Let's keep that!

Reply:
@tgallice @Miexil the "Congratulation" has always been really weird... 😅